docs/blockchain/contracts/nft-contract/nft-tables.experimental.md

```diff
@@ -0,0 +1,812 @@
+---
+title: 'NFT Tables'
+order: 1
+
+---
+
+# NFT Tables
+
+## factory.b
+
+-   Table: `factory.b`
+-   Code: `eosio.nft.ft`
+-   Scope: `eosio.nft.ft`
+-   Key: `id`
+
+The table contains uniq factories settings and the operational info.
+
+| Fields                  | Type                              | Description                                                                                                                             |
+| ----------------------- | --------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------- |
+| id                      | uint64_t                          | (primary key) The Uniq factory ID                                                                                                       |
+| asset_manager           | eosio::name                       | Account that manages the Uniq lifecycle - issuing, burning, reselling etc.                                                              |
+| asset_creator           | eosio::name                       | Account that ceates the Uniq factory.                                                                                                   |
+| minimum_resell_price    | eosio::asset                      | A minimum price when resell on marketplaces.                                                                                            |
+| resale_shares           | std::vector\<eosio::resale_share> | A vector of [account, share] pairs setting the share each account receives during the Uniq resale.                                      |
+| mintable_window_start   | std::optional\<uint32_t>          | The beginning of the time window when tokens can be minted.                                                                             |
```

Comment:
The term "token(s)" is used in several places, which should be replaced with "Uniq(s)"

Reply:
updated